lightning/src/blinded_path/message.rs

```diff
@@ -78,16 +79,61 @@ impl Writeable for ReceiveTlvs {
 	fn write<W: Writer>(&self, writer: &mut W) -> Result<(), io::Error> {
 		// TODO: write padding
 		encode_tlv_stream!(writer, {
-			(6, self.path_id, option),
+			(65537, self.context, option),
 		});
 		Ok(())
 	}
 }
 
+/// Represents additional data included by the recipient in a [`BlindedPath`].
+///
+/// This data is encrypted by the recipient and remains invisible to anyone else.
+/// It is included in the [BlindedPath], making it accessible again to the recipient
+/// whenever the [BlindedPath] is used.
```

Comment:
Use ticks around `BlindedPath` so it is formatted.

Reply:
Done! [8137b53](https://github.com/lightningdevkit/rust-lightning/commit/8137b53b96d4e7104bde8533b50d622230751ba2)
Thanks!